internal/generator/generator.go

```diff
@@ -22,6 +22,26 @@ import (
 //go:embed templates/options.go.tpl
 var templates embed.FS
 
+var templateImports = []string{
```

Comment:
At first sight - that is unnecessary, and you can leave this right in the tpl file. options-gen will invoke the imports.Process for code formatting. But maybe I do not see something?

Reply:
Yes, I was surprised, but I received conflicts with "errors" and with "fmt" twice in different cases.
Currently I cannot reproduce it, lol.

Remove.